roles/userspace-cni-install/tasks/vpp-install.yml

```diff
@@ -27,6 +27,14 @@
   when: ansible_os_family == "RedHat"
   register: vpp_sh_rh
 
+- name: write sys config with hugepages and memory settings
```

Comment:
I'm not entirely sure here, but the file (re)write might have to be done after VPP is installed, to avoid the file being modified again.

Reply:
moved